app/views/dashboard/index.html.erb

```diff
@@ -1,38 +1,14 @@
-<%= c 'hero' %>
-
-<section>
-  <%= c 'box', style: :large do %>
-    <%= c 'wrapper', style: :narrow do %>
-      <%= c 'stack', space: :large do %>
-      
-        <%= c 'heading', tag: :h2 do %>
-          <%= I18n.t 'dashboard.how_it_works' %>
-        <% end %>
-        
-        <%= c 'icon_list',
-          elements: I18n.t('dashboard.how_it_works_list'),
-          styles: [:vertical, :alternating]
-        %>
-
+<div class='Dashboard'>
```

Comment:
I received feedback from @tillprochaska that we should change the background-color of the dashboard page so I added this wrapping `div` and gave it styles, but this violates the pattern of component-only pages and introduces import views css and I wrote `div` that this bigger issue for me is the fact that we are not using semantic HTML.

> I added a commit for this, but I'm not happy about the way I achieved it. I added a div to wrap the entire page and added a class/background-color to it.
It's actually one of my pet peeves, that I would like to discuss with you. I really do not like the patter of using the main tag in the layout. I understand that it means that we cannot forget to have a main tag on a page, but it means that if I have the need of a tag to wrap the entire page, I am left with using a non-semantic HTML tag, div , when in reality the semantic choice is main as it contains the main content of the page.

I could open a PR that introduces a component, which is the main content of the page and would move the styles out of the views into this component and also update the existing page views, which aren't many. However, I think we can do this in an improvement PR.

Reply:
yeah I agree here too that introducing a new dashboard component as the main content of the page seems to be the way to go. Would be great if you could open a PR! 👍

Reply:
I've created an issue, which will have a follow-up PR See [Create MainComponent](https://github.com/tactilenews/100eyes/issues/1457)